newton/tests/test_collision_cloth.py

```
@@ -800,6 +802,273 @@ def test_mesh_ground_collision_index(test, device):
+    max_query_radius: float,
+    min_query_radius: float,
+    pos: wp.array(dtype=wp.vec3),
+    ref_pos: wp.array(dtype=wp.vec3),
+    edge_indices: wp.array(dtype=wp.int32, ndim=2),
+    edge_colliding_edges_offsets: wp.array(dtype=wp.int32),
+    edge_colliding_edges_buffer_sizes: wp.array(dtype=wp.int32),
+    edge_edge_parallel_epsilon: float,
+    # outputs
+    edge_colliding_edges: wp.array(dtype=wp.int32),
+    edge_colliding_edges_count: wp.array(dtype=wp.int32),
+    edge_colliding_edges_min_dist: wp.array(dtype=float),
```

Ruff (0.14.8)

Unused function argument: `edge_colliding_edges_min_dist`

(ARG001)